pkg/userdata/flatcar/provider.go

```diff
@@ -203,6 +205,17 @@ systemd:
             Environment=ALL_PROXY={{ .HTTPProxy }}
 {{- end }}
 
+    - name: containerd.service
+      dropins:
+        - name: 10-custom.conf
+          contents: |
+            [Service]
+            Restart=always
+            EnvironmentFile=-/etc/environment
+            Environment=CONTAINERD_CONFIG=/etc/containerd/config.toml
+            ExecStart=
+            ExecStart=/usr/bin/env PATH=${TORCX_BINDIR}:${PATH} ${TORCX_BINDIR}/containerd --config ${CONTAINERD_CONFIG}
```

Comment:
Since the `/etc/containerd/config.toml` isn't actually created anywhere, this won't start